go/common/host/host.go

```diff
@@ -23,8 +23,10 @@ type Host interface {
 	SubmitAndBroadcastTx(encryptedParams common.EncryptedParamsSendRawTx) (common.EncryptedResponseSendRawTx, error)
 	// ReceiveTx processes a transaction received from a peer host.
 	ReceiveTx(tx common.EncryptedTx)
-	// ReceiveBatch processes a batch received from a peer host.
-	ReceiveBatch(batch common.EncodedBatch)
+	// ReceiveBatches receives a set of batches from a peer host.
+	ReceiveBatches(batches common.EncodedBatches)
+	// SendBatches sends a set of batches to a peer host. Used during catch-up.
+	SendBatches(batchRequest common.EncodedBatchRequest)
```

Comment:
This API feels a little odd, one of these methods is driven from internally, one from externally so they don't feel like they belong on the same interface. I guess we should have P2PReceiver and P2PRequester interfaces or something but that can be a future refactor.